deployment/flatpak/about.md

```diff
@@ -0,0 +1,11 @@
++---
++title: Flatpak
++subsection: flatpak
++section: deployment
++description: >
++Flatpak (formerly xdg-app) is a software utility for software deployment, package management and application virutalization. It provides a sandbox environment in which users can run applications in isolation from the rest of the system.
++---
++
++# {{page.title}}
++
++[Flatpak](https://flatpak.org/) is a software utility for software deployment, package management and application virtualization. It provides a sandbox environment in which users can run applications in isolation from the rest of the system.
```

Comment:
Please create something like `References` sub-header (double hashtag - `## References`) with a link to documentation and you should provide at least basic usage, including installation (is there something you need to install to use Flatpacks on Fedora?) and something about where-to-find-them etc..

Description is not nearly enough I'd say.